com.unity.render-pipelines.high-definition/Documentation~/Override-Physically-Based-Sky.md

```diff
@@ -33,11 +33,16 @@ To change how much the atmosphere attenuates light, you can change the density o
 
 [!include[](snippets/Volume-Override-Enable-Properties.md)]
 
+### Model
+
+| **Property**                   | **Description**                                         |
+| ------------------------------ | ------------------------------------------------------- |
+| **Type**                       | Indicates a preset HDRP uses to simplify the Inspector. Selecting Earth will only show properties suitable to simulate Earth. |
```

Comment:
Indicates a preset HDRP uses to simplify the Inspector. If you select **Earth (Simple)** or **Earth (Advanced)**, the Inspector only shows properties suitable to simulate Earth.